00-Login-Kt/app/src/main/java/com/auth0/sample/MainActivity.kt

```diff
@@ -52,13 +83,16 @@ class MainActivity : AppCompatActivity() {
                 }
 
                 override fun onSuccess(credentials: Credentials?) {
+                    cachedCredentials = credentials!!
```

Comment:
check locally but if cachedCredentials is already optional, you probably don't need the 🔫 🔫 

```suggestion
                    cachedCredentials = credentials
```